component-library/component-lib/src/lib/header/header.component.html

```diff
@@ -1,6 +1,6 @@
 <header class="heading">
     <div class="grid-container" [id]="id">
-        <img src="{{imageURL}}">
+        <img src="{{imageURL}}" attr.aria-label="{{ 'Header.header-img-alt-text' | translate }}">
```

Comment:
Can't do this - don't want to use the translate pipe with static text in the library for a few reasons. See other text examples in the header

Reply:
will take a look.

Reply:
Fixed.